addon/application.go

```diff
@@ -122,16 +183,13 @@ func (h *AppTags) Replace(ids []uint) (err error) {
 		return
 	}
 	path := Params{api.ID: h.appId}.inject(api.ApplicationTagsRoot)
-	query := []Param{}
-	if h.source != nil {
```

Comment:
**hint**: removed redundant check.  Method returns at the beginning if h.source == nil.